packages/esm-service-queues-app/src/createDashboardLink.component.tsx

```diff
@@ -22,13 +25,25 @@ function DashboardExtension({ dashboardLinkConfig }: { dashboardLinkConfig: Dash
   }, [location.pathname]);
 
   return (
-    <ConfigurableLink
-      className={classNames('cds--side-nav__link', {
-        'active-left-nav-link': navLink.match(name),
-      })}
-      to={`${spaBasePath}/${name}`}>
-      {t('serviceQueues', 'Service queues')}
-    </ConfigurableLink>
+    <div>
```

Comment:
Is the `div` wrapper necessary here?

Reply:
@CynthiaKamau Please remove this div unless it is necessary for some reason.